tests/evm/test_mulmod.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from typing import Optional
+from zkevm_specs.evm import (
+    ExecutionState,
+    StepState,
+    verify_steps,
+    Tables,
+    Block,
+    Bytecode,
+    RWDictionary,
+)
+from zkevm_specs.util import rand_fq, RLC
+
+MAXU256 = (2**256) - 1
+
+
+TESTING_DATA = [
```

Comment:
Try adding `(MAXU256, MAXU256, 1)`...

Reply:
Well spotted!
I think rather than increasing the  range of `k` up to `MAXU256^2` we could do a previous step, reducing one of the factors mod `n`. This way `k` won't overflow:

$$
k = \frac{(a \text{ mod } n) \cdot b }{n } \leq \frac{ (n-1) \cdot b}{n} < b \leq MAXU256 
$$

Reply:
Included!